src/lib/test/fetchAccount.test.js

```diff
@@ -0,0 +1,84 @@
+import { describe, it, expect, vi } from 'vitest';
+import { fetchAccount, fetchAccountBalances } from '../stellar/horizonQueries'; 
+// @ts-ignore
+import { Server, StrKey } from 'stellar-sdk';
+
+vi.mock('stellar-sdk', async (importOriginal) => {
```

Comment:
```suggestion
import { Horizon, StrKey } from '@stellar/stellar-sdk';

vi.mock('@stellar/stellar-sdk', async (importOriginal) => {
```

You'll want to use the (recently renamed) `@stellar/stellar-sdk` package instead. The current version also makes a distinction between the `Horizon.Server` and `rpc.Server` classes, so that may come into play when it comes to the mock server stuff. (the `rpc.Server` stuff is irrelevant to the `horizonQueries`, though)

Reply:
It looks like the `Horizon.Server` vs. `Server` class change will need to be made in a lot (all?) of the test files